tasks/system_probe.py

```diff
@@ -1535,3 +1535,12 @@ def test_microvms(
     ctx.run(
         f"cd ./test/new-e2e && go run ./scenarios/system-probe/main.go --name usama-saqib-test {go_args} --shutdown-period 720",
     )
+
+
+@task
+def generate_git_file_diff(ctx):
+    file_diff = ctx.run(
+        "git diff --name-only $(git merge-base $(inv release.get-release-json-value base_branch) HEAD) HEAD", hide=True
+    )
+    file_diff_lines = file_diff.stdout.replace("\n", "\\n")
```

Comment:
Why do you need to escape the newlines?

Reply:
I wrote about that in Additional Notes of the PR (dotenv doesn't handle multiline very well so a way to bypass that is to escape newline to make a one liner, once it get's send it will still get correctly iterated over by shell for loop). I'll add a comment about that